slackify-html.js

```diff
@@ -38,6 +38,96 @@ function walkList(dom, ordered, nesting, start) {
             // Add the text content of the p element
             out += walkList(el.children, ordered, nesting) + "\n";
             break;
```

Comment:
everything you see below is just a copy of the `walk` function that does this element checking. The reason I have to add this into my `walkList` function is because when we are in a `<ul> or <ol>` we stay in this function and traverse the children recursively until we are done rather than before we went to the `walk` function